cms/check_number_of_missing_files_per_rse

```diff
@@ -21,35 +21,66 @@
 Probe to check the number of expected files per rse.
 """
 
-from __future__ import print_function
-
+import json
 import sys
-from datetime import datetime
+import traceback
+
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+from sqlalchemy import func
 from rucio.db.sqla.constants import RuleState
-from rucio.db.sqla.constants import RequestType
 from rucio.core import monitor
+from rucio.db.sqla.constants import RequestType
 from rucio.db.sqla import models
 from rucio.db.sqla.session import get_session
-from sqlalchemy import func
+from rucio.common.config import config_get
+
+PROM_SERVERS = config_get('monitor', 'prometheus_servers', raise_exception=False, default='')
+if PROM_SERVERS != '':
+    PROM_SERVERS = PROM_SERVERS.split(',')
+else:
+    PROM_SERVERS = None
+
+prom_labels_config = config_get('monitor', 'prometheus_labels', raise_exception=False, default='{}')
+extra_prom_labels = json.loads(prom_labels_config)
 
 # Exit statuses
 OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
 
 if __name__ == '__main__':
-    status = OK
-    session = get_session()
+    registry = CollectorRegistry()
+    labelnames = ['rse']
+    labelnames.extend(extra_prom_labels.keys())
+    missing_files_gauge = Gauge('rucio_judge_number_of_missing_files','Number of missing files in a RSE',
+                             labelnames=labelnames, registry=registry)
+    replicating_files_gauge = Gauge('rucio_judge_number_of_replicating_files', 'Number of files being replicated to a RSE',
+                                labelnames=labelnames, registry=registry)
+    #status = OK
```

Comment:
I don't think there is any reason to change the exit codes from what they originally were